modules/selecting-cluster-capabilities.adoc

```diff
@@ -12,11 +12,22 @@ During a customized installation, you create an `install-config.yaml` file that
 [source,yaml]
 ----
 capabilities:
-  baselineCapabilitySet: v4.13 <1>
+  baselineCapabilitySet: v4.11 <1>
   additionalEnabledCapabilities: <2>
   - CSISnapshot
   - Console
   - Storage
 ----
-<1> Defines a baseline set of capabilities to install. Valid values are `None`, `v4.12`, `v4.13`, and `vCurrent`. If you select `None`, all optional capabilities will be disabled. The default value is `vCurrent`, which enables all optional capabilities.
+<1> Defines a baseline set of capabilities to install. Valid values are `None`, `vCurrent` and `v4.x`. If you select `None`, all optional capabilities will be disabled. The default value is `vCurrent`, which enables all optional capabilities.
 <2> Defines a list of capabilities to explicitly enable. These will be enabled in addition to the capabilities specified in `baselineCapabilitySet`.
++
+[NOTE]
+====
+`v4.x` refers to any value up to and including the current cluster version. 
+For example, valid values for OpenShift Container Platform 4.12 cluster are `v4.11` and `v4.12`. Similarly, valid values for OpenShift Container Platform 4.13 cluster are `v4.11`, `v4.12`, and `v4.13`. 
```

Comment:
```suggestion
For example, valid values for {product-title} 4.12 cluster are `v4.11` and `v4.12`. Similarly, valid values for {product-title} 4.13 cluster are `v4.11`, `v4.12`, and `v4.13`. 
```

Reply:
Also, I do not think this sentence will work. 4.13 is not released yet and if we CP back to 4.12, those users might be confused. I am not sure what a solution to this would be though unless you leave the 4.12 example and remove the 4.13 one maybe?

Reply:
Also, I think this note would fit better under the first footnote.

Reply:
I will remove the 4.13 example. Accordingly, I will move this example under the first footnote.